### ISSUE TITLE
Make more things configurable in Preferences

### ISSUE THREAD

**Issue by infotexture** [artynusov/MacTimeLog#3]
- Specify log file location.  
  (use something other than `~/Library/Application Support/MacTimeLog/log.txt`)
- Honor system date & time formats from _Language & Text_ prefs.  
  Allow user to select `[short|medium|long|full]` formats in _MacTimeLog_ prefs.  
  (For example, use **short** (ISO-8601) format in Log, but _"Your working day started on **full**"_.)

**Comment by artynusov** [comment]
1. Do you want to use multiple logs? Or do you want it to be easy accessible ?
2. This is good idea, need to think about best way of implementing it.

Thank you for good suggestions!

**Comment by infotexture** [comment]
Regarding 1, I'm more interested in being able to store the file elsewhere _(perhaps for version control)_. I don't see an immediate need for multiple logs, but others might.

Regarding 2, I think it makes sense to use ISO 8601 as the _default_ format for timestamps in the log, since the `[YYYY]-[MM]-[DD] [hh]:[mm]` can be easily sorted, whereas the current default format cannot. However, I'm sure many users would appreciate the ability to change this setting, and since they've presumably already configured their system with their preferred settings, it would be nice to pick those up. 

Maybe add a pref that reads **Timestamp format:** `[short|medium|long|full]` with a link to the system _Language & Text_ prefs.

**Comment by artynusov** [comment]
1. Clear, makes more sense now.
2. Log already sorted, but you right, some users may want to use different format.

**Comment by infotexture** [comment]
Yes, _within MacTimeLog,_ entries are currently sorted as they are created, since lines are appended to the end of the file, **but** if entry lines are run through the UNIX `sort` command by an external process, the current date format yields undesirable results, as the lines are sorted by the days only, ignoring month, year, or time.

In this case,

```
12-31-2008 12:00   Client :: The First Task
01-01-2009 12:00   Client :: The Second Task
```

becomes:

```
01-01-2009 12:00   Client :: The Second Task
12-31-2008 12:00   Client :: The First Task
```

Whereas ISO 8601 `[YYYY]-[MM]-[DD] [hh]:[mm]` format sorts as expected.
